packages/desktop/src/renderer/components/Channel/ChannelComponent.tsx

```diff
@@ -241,6 +286,33 @@ export const ChannelComponent: React.FC<ChannelComponentProps & UploadFilesPrevi
             duplicatedUsernameModalHandleOpen={duplicatedUsernameModalHandleOpen}
           />
         </ChannelMessagesWrapperStyled>
+        <Grid
```

Comment:
I'm wondering if maybe all this new code in ChannelComponent could be a completely new component. I mean it has its own purpose and it can have its own separate place in storybook.
Also (for the sake of clarity) its props could be grouped in Channel.tsx similar to uploadFilesPreviewProps or fileActionsProps.

Reply:
Merged the code